### ISSUE TITLE
Usability error when a file in a suite cannot be found

### ISSUE THREAD

**Issue by SchumacherFM** [vdobler/ht#17]
I have a suite:

```
{
    Name: " Shop Basic Functionality",
    KeepCookies: true,

    Main: [
        {File: "homepage-basics.ht"}
    ]

    Teardown: [
        {File: "get-admin-token.ht"}
  ],
}
```

The file `homepage-basics.ht` exists but the file `get-admin-token.ht` does not exists.

When I run now:

```
$ ./ht/ht_darwin exec -output ./reports/ -Dfile ./env/local.json ./frontend/index.suite
```

The result is:

```
$ ./ht/ht_darwin exec -output ./reports/ -Dfile ./env/local.json ./frontend/index.suite
Seeding random number generator with 1507875229665009841.
Resetting global counter to 1.
Using "phantomjs" as PhantomJS executable.

Total 0,  Passed 0,  Skipped 0,  Errored 0,  Failed 0,  Bogus 0
NOTRUN
```

There is no indication that a file is missing or spelt wrong. You can search all over the suite and test files to look for the bug ... bit time consuming.

Would be nice to have an appropriate error message. 🙌

ht version: latest

**Comment by vdobler** [comment]
Confirmed.

**Comment by vdobler** [comment]
Fixed in 2f3a5ec
